application/notifications/messages.go

```diff
@@ -11,3 +16,24 @@ type Message struct {
 	ToPhone   string
 	Subject   string
 }
+
+// NewEmailMessage returns a message with the FromEmail, the Data.appName and Data.uiURL already set
+func NewEmailMessage() Message {
+	msg := Message{
+		FromEmail: domain.EmailFromAddress(nil),
+		Data: map[string]interface{}{
+			"appName": domain.Env.AppName,
+			"uiURL":   domain.Env.UIURL,
+		},
+	}
+	return msg
+}
+
+// sets the msg ToName and ToEmail based on the steward's information
+func (m *Message) AddToSteward() {
```

Comment:
```suggestion
func (m *Message) ToSteward() Message {
```
Then you can do this:
```
msg := notifications.NewEmailMessage().ToSteward()

Reply:
:+1: